static/app/components/events/interfaces/frame/stacktraceLink.spec.tsx

```diff
@@ -197,4 +200,42 @@ describe('StacktraceLink', function () {
       expect(container).toBeEmptyDOMElement();
     });
   });
+
+  it('renders the codecov link', async function () {
+    MockApiClient.addMockResponse({
+      url: `/projects/${org.slug}/${project.slug}/stacktrace-link/`,
+      body: {
+        config,
+        sourceUrl: 'https://github.com/username/path/to/file.py',
+        integrations: [integration],
+        codecovStatusCode: CodecovStatusCode.COVERAGE_EXISTS,
+      },
+    });
+    render(<StacktraceLink frame={frame} event={event} line="foo()" />, {
+      context: TestStubs.routerContext(),
+      organization: org,
+    });
+    expect(await screen.findByText('View Coverage Tests on Codecov')).toHaveAttribute(
+      'href',
+      'https://app.codecov.io/gh/path/to/file.py'
+    );
+  });
+
+  it('renders the missing coverage warning', async function () {
+    org.features = ['codecov-stacktrace-integration'];
```

Comment:
don't modify the org object here since it will leak into the next test
```suggestion
    const organization = {...org, features: ['codecov-stacktrace-integration']}
```